### PR TITLE
feat(queries): [Experimental] Allow customization of # of queries in Query tab via env var

### PR DESCRIPTION
**Pull request by gabe-lyons**
This feature is experimental and is subject to change / deprecation. In the next 6-12 months the query tab will likely see a number of changes around UX and usability. This env var provides a short term improvement for users who want to show more than 5 queries per entity.

Usage:
Set `REACT_APP_QUERIES_TAB_RESULT_SIZE` in your GMS container to be the number of queries you'd like to show.

CleanShot 2023-01-05 at 12 52 35

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-service/factories/src/main/resources/application.yml [hunk 1]
```diff
@@ -28,15 +28,14 @@ authentication:
...
-
   # Configurations for the default DataHub policies-based authorizer.
   defaultAuthorizer:
     enabled: ${AUTH_POLICIES_ENABLED:true}
     cacheRefreshIntervalSecs: ${POLICY_CACHE_REFRESH_INTERVAL_SECONDS:120}
 
 ingestion:
   enabled: ${UI_INGESTION_ENABLED:true}
-  defaultCliVersion: '${UI_INGESTION_DEFAULT_CLI_VERSION:@cliMajorVersion@}'
+  defaultCliVersion: "${UI_INGESTION_DEFAULT_CLI_VERSION:@cliMajorVersion@}"
```
Comment (aditya-radhakrishnan) [inline metadata-service/factories/src/main/resources/application.yml:38]: Is this change intended?

File: metadata-service/factories/src/main/resources/application.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ authentication:
 
   # Required if enabled is true! A configurable chain of Authenticators
   authenticators:
-      # Required for authenticating requests with DataHub-issued Access Tokens - best not to remove.
+    # Required for authenticating requests with DataHub-issued Access Tokens - best not to remove.
```
Comment (aditya-radhakrishnan) [inline metadata-service/factories/src/main/resources/application.yml:8]: Is this change intended?

**Review by aditya-radhakrishnan — APPROVED** [review]
LGTM but revert the formatting fixes in `application.yml`, unless they were intentional

File: metadata-service/factories/src/main/resources/application.yml (hunk 2 above)
Comment (gabe-lyons) [reply metadata-service/factories/src/main/resources/application.yml:8]: Automatic reformatting to be consistent with the rest of the file

File: metadata-service/factories/src/main/resources/application.yml (hunk 1 above)
Comment (gabe-lyons) [reply metadata-service/factories/src/main/resources/application.yml:38]: Automatic reformatting to be consistent with the rest of the file
